packages/fluentui/docs/src/components/Sidebar/Sidebar.tsx

```diff
@@ -559,6 +564,11 @@ const Sidebar: React.FC<RouteComponentProps & SidebarProps> = props => {
           setActiveItemIds(activeItemIds);
         }}
       />
+      <Flex column>
```

Comment:
@miroslavstastny the original PR disabled this part because: https://github.com/microsoft/fluentui/pull/13963/files#r494769209 

Should I leave it as disabled?

Reply:
Leave it disabled for now